### DISCUSSION TITLE
Do OPT managers finish early on Fridays?

### DISCUSSION THREAD

**Discussion by laat0003** [usgs/pestpp#313] [Q&A] [answered]
Howdy,

I'm running a stack based OPT problem. Worked flawlessly with a single model optimization and now I'm running with the stack. I'm passing in both par and obs stacks ( .0 stacks from previous OPT run) so it's only building the response matrix, which happens without incident. Then the manager saves the jcb and disappears to the pub for afternoon drinks. No entries in the coin log with verbosity on 4. The .log file stops after loading the obs stack, so I'm guessing there's something in there it doesn't like. Both the par and obs stacks are csvs that were created by OPT previously.
My guess is its late on a Friday afternoon and nobody is making this obs-snob of a manager work over the weekend.

Cheers,

**Comment by jtwhite79** [comment, marked as answer]
most pest++ codes finish early on Friday haha.  

There are several tests that (try to) cover the use of stacks with pestpp-opt and I poked around a lil this morning and everything seems to be working as expected for these tests.  Can you share these files with me so I can see why the master is exiting silently?

**Reply by laat0003** [reply]
Thanks for the prompt response. I can't seem to upload the par and obs stacks because they exceed 25MB and GitHub no-likey. I've sent you an invite to a repo where the other OPT output files are. Unsure if that even helps or what else you might need to troubleshoot.

**Reply by jtwhite79** [reply]
Lemme have a go with those files and see if I can trigger the issue...

**Reply by laat0003** [reply]
Thanks for that. If any other files are needed please let me know and I'll upload them to the repo ASAP.

**Reply by laat0003** [reply]
Thanks for the assist with that one. Updating to the pre-release version and doing the base_jacobian + hotstart_resfile + skip_final options got me rolling again. Had to adjust my constraints slightly to get a feasible solution but all running smoothly now.

**Reply by jtwhite79** [reply]
Sweet. Keep us posted on how it goes...
